core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/diskModels.scala

```diff
@@ -1,6 +1,55 @@
 package org.broadinstitute.dsde.workbench.leonardo
 
+import enumeratum.{Enum, EnumEntry}
+import org.broadinstitute.dsde.workbench.google2.{DiskName, ZoneName}
+import org.broadinstitute.dsde.workbench.model.google.GoogleProject
+
+final case class PersistentDisk(id: DiskId,
+                                googleProject: GoogleProject,
+                                zone: ZoneName,
+                                name: DiskName,
+                                googleId: Option[GoogleId],
+                                samResourceId: DiskSamResourceId,
+                                status: DiskStatus,
+                                auditInfo: AuditInfo,
+                                size: DiskSize,
+                                diskType: DiskType,
+                                blockSize: BlockSize,
+                                labels: LabelMap)
```

Comment:
Since we have a `Failed` disk status, should we have an `errors` field here?

Reply:
Yeah, I didn't make `errors` at all.. was thinking of punting that till we have more understanding of what errors can occur.